content/docs/reactivesearch/react-searchbox/apireference.md

```diff
@@ -24,120 +22,90 @@ Example uses of searchbox UI:
 -   Searching a rental listing by its `name` or `description` fields.
 -   Searching across e-commerce products.
 
-## Usage
-
-### Basic Usage
-
-```js
-<SearchBox
-	app="good-books-ds"
-	credentials="nY6NNTZZ6:27b76b9f-18ea-456c-bc5e-3a5263ebc63d"
-	dataField={['original_title', 'original_title.search']}
-/>
-```
-
-### Usage With All Props
-
-```js
-<SearchBox
-	app="good-books-ds"
-	appbaseConfig={{
-        recordAnalytics: true,
-        enableQueryRules: true,
-        userId: 'jon@appbase.io',
-        customEvents: {
-           platform: "ios",
-           device: "iphoneX"
-        }
-    }}
-	credentials="nY6NNTZZ6:27b76b9f-18ea-456c-bc5e-3a5263ebc63d"
-	dataField={[
-		{ field: 'original_title', weight: 1 },
-		{ field: 'original_title.search', weight: 3 },
-	]}
-	title="Search"
-	defaultValue="Songwriting"
-	placeholder="Search for books"
-	autosuggest={true}
-	defaultSuggestions={[
-		{ label: 'Songwriting', value: 'Songwriting' },
-		{ label: 'Musicians', value: 'Musicians' },
-	]}
-	size={10}
-	highlight={true}
-	highlightField="group_city"
-	queryFormat="or"
-	fuzziness="AUTO"
-	showClear
-	showVoiceSearch
-/>
-```
-
-## Props
-
--   **app** `string` [required]
+This library has 3 essentials parts
+
+-   **SearchBase**: It is used as an internal dependency to connect with an Elastic Search system to perform search queries)
+-   **SearchBox**: The core of the library which helps with data access and manipulation with any UI components
```

Comment:
The definition is not correct. It should be described as a search input component to perform. For ref. check the `DataSearch` docs.